### PR TITLE
allowFontScaling and NavigationBar title/right button fix

### PR DESCRIPTION
**Pull request by yamill**
<h2>allowFontScaling fix <Text></h2>
This is to fix the default font scaling property `allowFontScaling`. In this commit 53fb5b6, it says that the default should be `false`. Right now it's set to `true`. Not every app wants their font sizes to be adjusted, unless it's some sort of reading app.

<h2>NavigationBar Title fix</h2>
This is to fix the NavigationBar title. It is not being is not being centered properly in Landscape mode.

This is because of commit dbe8e31c209dd1c4a8f48c5e26cbb67edb30d2a7.

Changes were made to remove unnecessary use of absolute screen width, but looks like `Title:` was left out of the equation.

<h2>NavigationBar RightButton fix</h2>

The RightButton was being fixed to the screen width causing it to sit a little passed halfway in Landscape.

To fix this, I removed the `width` and `left` properties and added the `right` property to equal to 0.

This is because of commit dbe8e31c209dd1c4a8f48c5e26cbb67edb30d2a7.

### CODE REVIEW

File: Libraries/Text/Text.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ var viewConfig = {
   validAttributes: merge(ReactNativeViewAttributes.UIView, {
     isHighlighted: true,
     numberOfLines: true,
-    allowFontScaling: true,
+    allowFontScaling: false,
```
Comment (ide) [inline Libraries/Text/Text.js]: ? i don't think this does what you meant

Comment (yamill) [inline Libraries/Text/Text.js]: The commit that changed this said the property should be defaulted to `false`

Comment (yamill) [inline Libraries/Text/Text.js]: @ide I updated my description above. Sorry for the confusion.

Comment (ide) [inline Libraries/Text/Text.js]: what i mean is that validAttributes doesn't control the default value of the props, which you've already done below.

Comment (yamill) [inline Libraries/Text/Text.js]: Gotcha! I just made a commit and reverted that. Thanks.

**Comment by brentvatne** [comment]
Hi there @yamill!

Could you split this into two pull requests? I'm happy to accept the font scaling fix immediately but I don't know enough to accept the NavigationBar fix. Also, in the interest of keeping our history clean, could you just have a single commit in each of those pull requests?

Thanks! :smile: :tada:

**Comment by yamill** [comment]
@brentvatne ok awesome will do. thanks! :smile:

**Pull request closed without merging**